modules/ROOT/pages/changelog.adoc

```diff
@@ -2,6 +2,21 @@
 
 This page lists changes to the {product-name}.
 
+== Version 5.1.17
+
+[cols="1,2", options="header"]
+|===
+| Feature | Details
+
+a|
+label:bug[]
+label:fixed[]
+
+Fix CDC polling to prevent repeated scans of transaction log.
+| When CDC source connector is configured with specific patterns and no changes are detected, the connector uses the stored offset to re-scan the same transaction log entries in the next poll cycle.
+This has been fixed so that the connector now correctly advances the in memory offset even when no changes are detected.
```

Comment:
```suggestion
| When the CDC source connector is configured with specific patterns and no changes are detected, the connector uses the stored offset to re-scan the same transaction log entries in the next poll cycle.
This has been fixed so that the connector now correctly advances the in-memory offset even when no changes are detected.
```